calendar_resource/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2010-2014 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Calendar Resources",
+    "summary": "New features to facilitate resource management with meetings.",
+    "version": "10.0.1.0.0",
+    "category": "CRM",
+    "website": "http://www.savoirfairelinux.com",
+    "author": "Savoir-faire Linux",
```

Comment:
We probably contributed enough for co-author. Also need to add `Odoo Community Association (OCA)`